### DISCUSSION TITLE
Compatibility for different OPC Clients

### DISCUSSION THREAD

**Discussion by stefanadam** [BiancoRoyal/node-red-contrib-iiot-opcua#144] [Q&A]
Hi there, new to this and trying out this great OPC server you build. Seems to work splendid.

Initially I had issues connecting to it, when using UaExpert it seems to connect without any issues, but when I tried Matrikon OPC UA Explorer (free) I get error messages, something to do with the server location (my computer and the OPC server are on a different network), but accessible via the IP address.
I tried all kind of different settings, but could not get it to work.
Is this something that is in the OPC specifications that is missing or something I have to set up inside the Address Space?

**Comment by ghost** [comment]
Hello,
could you share the error logs/messages Matrikon gives you, and do you get any errors from Node-RED? 

The IIoT-Server node sets up the Address Space by default, but the IIoT-Flex-Server node requires a script that sets up the Address Space.

Given your description of the problem, and that UaExpert does work, it seems likely to me that this is an issue on Matrikon's end, but without more details, I can't say for sure.
